src/lelantus.cpp

```diff
@@ -698,18 +704,6 @@ bool CheckLelantusTransaction(
 {
     Consensus::Params const & consensus = ::Params().GetConsensus();
 
-
-    if(tx.IsLelantusJoinSplit()) {
-        CAmount nFees;
-        try {
-            nFees = lelantus::ParseLelantusJoinSplit(tx)->getFee();
```

Comment:
Is this set anywhere else, since it's now removed?

Reply:
nFees value is not used in this context, it was being used previously, and missed to remove this during refactorin

Reply:
Was this resolved?

Reply:
Relying on @levonpetrosyan93's comment that it is unused.